doc/syntax.md

```diff
@@ -53,45 +334,84 @@ end
   `<name>`. The content will be translated into a single neuron model
   that can be instantiated in NEST using `nest.Create(<name>)`. This
   is the top-level block of a model specification and all following
-  blocks are contained in this block.
-
-`parameters`
-: blabla
-
-`state`
-: blabla
-
-`internals`
-: blabla
-
-`equations`
-: blabla
-
-`input`
-: blabla
-
-`output`
-: blabla
-
-`update`
-: blabla
-
-`function`
-: blabla
+  blocks are contained in this block. The block is closed with the `end` keyword.
+
+`parameters:`
+:  This block is composed of a list with variable declarations. This block is
+supposed to contain variables
+which remain constant during the concrete simulation, but they can vary among
+different simulations or instantiations neuron of the same neuron. The block
+is closed with the `end` keyword.
+
+`state:`
+: This block is composed of a list with variable declarations. These variables are supposed to
+model the time dependent  state of the neuron. Only variables form this block can be refined with
+differential equations. The block terminates with the `end` keyword.
+
+`internals:`
+:  This block is composed of a list with variable declarations. These variables
+are supposed to be constant during the simulation run. Therefore, its initialization
+expression can only reference parameter- or another internal-variables. The block
+is closed with the `end` keyword.
+
+`equations:`
+: This block is composed of shape definitions and differential equations. In will
+be explained in further details later in the manual. The block is closed with
+the `end` keyword.
+
+`input:`
+: The block is composed of a list of input ports. In will
+be explained in further details later in the manual. The block terminates with the `end` keyword.
+
+`output:`
+: Defines which kinds of events the neuron can fire. Currently, only `spike` is
+supported. No `end` is necessary at the end of the block.
+```
+output: spike
+```
+`update:`
+: Inside the block arbitrary code in the introduced programming language can be
+implemented. The block terminates with the `end` keyword.
+
```

Comment:
Also explain that this block is the central piece of code for updating the dynamics of the neuron and that the pre-defined variable `t` is available here and that the block is executed in the update loop of NEST.